app/text_extractor.py

```diff
@@ -8,6 +8,8 @@
 from pdf2image import convert_from_path
 from PIL import Image, ImageEnhance
 
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
```

Comment:
I would not encourage to hardcode the executable's path on the code due to portability issue. Kindly please refer to the pytesseract installation link on ReadMe.md. Do let me know if you have troubles running the app after the installation.